staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go

```diff
@@ -38,16 +39,10 @@ func LoadPolicyFromFile(filePath string) (*auditinternal.Policy, error) {
 		return nil, fmt.Errorf("failed to read file path %q: %+v", filePath, err)
 	}
 
-	policyVersioned := &auditv1alpha1.Policy{}
-
-	decoder := audit.Codecs.UniversalDecoder(auditv1alpha1.SchemeGroupVersion)
-	if err := runtime.DecodeInto(decoder, policyDef, policyVersioned); err != nil {
-		return nil, fmt.Errorf("failed decoding file %q: %v", filePath, err)
-	}
-
 	policy := &auditinternal.Policy{}
-	if err := audit.Scheme.Convert(policyVersioned, policy, nil); err != nil {
-		return nil, fmt.Errorf("failed converting policy: %v", err)
+	decoder := audit.Codecs.UniversalDecoder(auditv1beta1.SchemeGroupVersion, auditv1alpha1.SchemeGroupVersion)
```

Comment:
Please take a look at here.
Now policy file support both alpha and beta versions.
But users `must` edit policy file in this format:
```
apiVersion: audit.k8s.io/v1alpha1    # this line is necessary!
kind: Policy
rules:
  - level: None
```